packages/e2e-tests/specs/performance/site-editor.test.js

```diff
@@ -104,13 +121,22 @@ describe( 'Site Editor Performance', () => {
 				firstBlock,
 			} = await getLoadingDurations();
```

Comment:
I guess same as the previous comment here.

Reply:
I was originally planning to keep things as close as possible between throwaway and real runs, but it doesn't really matter if the throwaway runs don't read the metrics. Moved.